aten/src/ATen/native/Resize.cpp

```diff
@@ -10,4 +10,8 @@ Tensor& resize_cpu_(Tensor& self, IntArrayRef size) {
   return self;
 }
 
+Tensor& resize_as_cpu_(Tensor& self, const Tensor& the_template) {
+  return resize_cpu_(self, the_template.sizes());
```

Comment:
does this happen to fix https://github.com/pytorch/pytorch/issues/11665?

Reply:
```python
In [1]: torch.randn(3).resize_as_(torch.ones(2, dtype=torch.uint8))                                                                            
Out[1]: tensor([-0.5847,  1.3565])
``` 
after porting